Examples/ImageGridManipulation/Documentation.rst

```diff
@@ -0,0 +1,60 @@
+Image Grid Manipulation
+=======================
+
+
+Overview
+--------
+
+There are numerous SimpleITK filters that have similar functions, but very
+important differences. The filters that will be compared are:
+
+ - ``JoinSeriesImageFilter()`` - Joins multiple N-D images into an (N+1)-D  image
+ - ``ComposeImageFilter()`` - Combines several scalar images into a  multicomponent vector image
+ - ``VectorIndexSelectionCastImageFilter()`` - Extracts the selected index of  the input pixel type vector (the input image pixel type must be a vector and the output a scalar)
+ - ``ExtractImageFilter()`` - Crops an image to the selected region bounds  using vectors; Collapses dimensions unless dimension is two
+ - ``CropImageFilter()`` - Similar to ``ExtractImageFilter()``, but crops an  image by an ``itk::Size`` at upper and lower bounds
+ - Image Slicing Operator [only in Python] - Uses pythonic slicing (``img[i:j, k:l]``) to extract a subregion of an image
+
+All of these operations will maintain the physical location of the pixels,
+instead modifying the image's metadata.
+
+Comparisons
+-----------
+
+Composition Filters
++++++++++++++++++++
+While ``JoinSeriesImageFilter()`` merges multiple images of the same pixel
+type in N dimensions into an image in N+1 dimensions, ``ComposeImageFilter()``
+will combine scalar images into a vector image of the same dimension. The
+former is useful for connecting a series of contiguous images while the latter
+is more useful for merging multiple channels of the same object into one image
+(such as RGB).
+
+Extraction Filters
+++++++++++++++++++
+``VectorIndexSelectionCastImageFilter()`` will isolate a single channel in a
+vector image and return a scalar image. On the other hand,
+``ExtractImageFilter()`` and ``CropImageFilter()`` will extract and return a
+subregion of an image, using an ExtractionRegion size and index and
+``itk::Size``'s respectively. However, note that only the
+``ExtractImageFilter()`` collapses dimensions. The Python image slicing
```

Comment:
Remove reference to Python as it also works in R, just say "The image slicing..." .